client/blocks/app-banner/index.jsx

```diff
@@ -69,6 +71,18 @@ export class AppBanner extends Component {
 		} else {
 			this.state = { isDraftPostModalShown: false };
 		}
+
+		this.loadExperiment();
+	}
+
+	loadExperiment() {
+		// Set a default value just in case the assignment hasn't loaded yet
+		this.experimentIsControl = true;
+
+		loadExperimentAssignment( APP_BANNER_EXPERIMENT_NAME ).then( ( experimentObject ) => {
+			const variationName = experimentObject.variationName;
+			this.experimentIsControl = variationName == null;
```

Comment:
I think this would be better to change things to 
```
this.experimentIsControl = ! experimentObject?.variationName;
```

Since `experimentObject.variationName` could also be undefined.

Reply:
Nice, I like this. Done in: 6e21ce2290e0a06b646dcf56bb8a1ed60f956ffb